test/math/FloatPoint2DTest.ooc

```diff
@@ -49,8 +50,8 @@ FloatPoint2DTest: class extends Fixture {
 		this add("casting", func {
 			value := "10.00, 20.00"
 			expect(this point3 toString(), is equal to(value))
-			expect((FloatPoint2D parse(value)) x, is equal to((this point3) x))
-			expect((FloatPoint2D parse(value)) y, is equal to((this point3) y))
+			expect((FloatPoint2D parse(Text new(value))) x, is equal to((this point3) x))
+			expect((FloatPoint2D parse(Text new(value))) y, is equal to((this point3) y))
```

Comment:
This "fix" of mine is a smaller change than the one you already implemented, which is why I suggest it now.